server/src/internalClusterTest/java/org/opensearch/gateway/RecoveryFromGatewayIT.java

```diff
@@ -734,4 +740,97 @@ public void testMessyElectionsStillMakeClusterGoGreen() throws Exception {
         internalCluster().fullRestart();
         ensureGreen("test");
     }
+
+    public void testSingleShardFetch() {
```

Comment:
please suffix UsingBatchAction in all the new tests.

Reply:
Added